RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -476,7 +480,7 @@ extension ProfileStore {
 		// We present an alert to user where they must choice if they wanna keep using Profile
 		// on this device or delete it. If they delete a new one will be created and we will
 		// onboard user...
-		let choiceByUser = await overlayWindowClient.scheduleAlertAwaitAction(.profileUsedOnAnotherDeviceAlert(
+		let choiceByUser = await overlayWindowClient.scheduleAlert(.profileUsedOnAnotherDeviceAlert(
```

Comment:
Why is this still here? do we have the claiming in two places? If still needed, we should present the new claim full screen, not alert.

Reply:
This particular piece of code is definitely not needed, but I want to go through ProfileStore with Cyon before cleaning it up, there is some logic that might still be useful.